var/spack/repos/builtin/packages/gnutls/package.py

```diff
@@ -17,15 +17,21 @@ class Gnutls(AutotoolsPackage):
     homepage = "http://www.gnutls.org"
     url      = "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.5/gnutls-3.5.19.tar.xz"
 
+    version('3.6.7.1', sha256='881b26409ecd8ea4c514fd3fbdb6fae5fab422ca7b71116260e263940a4bbbad')
     version('3.5.19', sha256='1936eb64f03aaefd6eb16cef0567457777618573826b94d03376bb6a4afadc44')
     version('3.5.13', '4fd41ad86572933c2379b4cc321a0959')
     version('3.5.10', '336c03a71ba90184ffd0388075dde504')
     version('3.5.9',  '0ab25eb6a1509345dd085bc21a387951')
     version('3.3.9',  'ff61b77e39d09f1140ab5a9cf52c58b6')
 
+    def url_for_version(self, version):
+        url = "https://www.gnupg.org/ftp/gcrypt/gnutls/v{0}/gnutls-{1}.tar.xz"
+        return url.format(version.up_to(2), version)
```

Comment:
This package already has a `url_for_version` defined.

Reply:
My bad. I've removed the duplicate `url_for_version` in the latest commit.